core/src/main/java/hudson/slaves/CloudSlaveRetentionStrategy.java

```diff
@@ -38,7 +38,7 @@ public long check(T c) {
                 }
             }
         }
-        return checkCycle();
+        return TimeUnit.MILLISECONDS.toMinutes(checkCycle());
```

Comment:
According to the docs https://github.com/jenkinsci/jenkins/blob/ece17e043101dba0656a12266378e80328dd2592/core/src/main/java/hudson/slaves/RetentionStrategy.java#L61-L62

This change is correct